_less/screen.less

```diff
@@ -907,9 +907,11 @@ table td,table th{
 	border:2px solid #ee9209;
 	color:#fff;
 }
-#glossary_term {
+.glossary_term {
 	display: block;
 	width: 20em;
+	text-align: center;
```

Comment:
This seems to center the text in the search box, which looks weird to me.  I think Google and every other search box I've ever used was left-aligned.

![2015-04-09-144534_1280x800_scrot](https://cloud.githubusercontent.com/assets/61096/7074292/4cc7569a-dec7-11e4-9f77-f4be01184358.png)

How do you feel about me removing this one line when I merge?